CMakeLists.txt

```diff
@@ -31,6 +31,19 @@ message(STATUS "Using protobuf ${Protobuf_VERSION}")
 find_package(gRPC CONFIG REQUIRED)
 message(STATUS "Using gRPC ${gRPC_VERSION}")
 
+# OpenVINO
+include_directories(
+    ${OpenCV_INCLUDE_DIRS}
+    /opt/intel/openvino_2023/runtime/include
+    /opt/intel/openvino_2023/runtime/include/ie
+    /opt/intel/openvino_2023/runtime/include/ngraph
+)
```

Comment:
Use [`target_include_directories`](https://cmake.org/cmake/help/latest/command/target_include_directories.html) instead and specify the required directory for each target. Mark as `PRIVATE` if the headers are only included in the source files, and mark as `PUBLIC` if the headers are also included in the header files.